plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -303,13 +414,17 @@ protected boolean canHandle(Network network, Service service) {
         }
 
         if (!_networkModel.isProviderForNetwork(getProvider(), network.getId())) {
-            s_logger.debug("NuageElement is not a provider for network " + network.getDisplayText());
+            if (s_logger.isDebugEnabled()) {
+                s_logger.debug("NuageVsp is not a provider for network " + network.getDisplayText());
```

Comment:
This code is just checking whether NuageVsp is included as provider for the given network, no need for specific information about the Nuage VSP.